src/testUtils/mocks/entities.ts

```diff
@@ -1645,6 +1676,38 @@ function initDividendDistribution(opts?: DividendDistributionOptions): void {
   configureDividendDistribution(dividendDistributionOptions);
 }
 
+/**
+ * @hidden
+ * Configure the Agent instance
+ */
+function configureAgent(opts: AgentOptions): void {
+  const agent = ({
+    did: opts.did,
+    ticker: opts.ticker,
+    getPermissionGroup: agentGetPermissionGroupStub.resolves(opts.getPermissionGroup),
+  } as unknown) as MockAgent;
+
+  Object.assign(mockInstanceContainer.agent, agent);
+  agentConstructorStub.callsFake(args => {
+    const value = merge({}, agent, args);
+    Object.setPrototypeOf(value, require('~/internal').Agent.prototype);
```

Comment:
Agent extends Identity, and it should be reflected here. Check how it's done for NumberedPortfolio